_posts/2022-09-06-conscryptmode.md

```diff
@@ -0,0 +1,19 @@
+Google Summer of Code (GSoC) is a program where external participants can contribute to an open source project over a few months. We learn new computer science concepts, how to work on an open source repository, and create real contributions to Google code!
+
+My project was `Switching Robolectric from BouncyCastle to Conscrypt as the default security provider`. Robolectric was using [BouncyCastle](https://www.bouncycastle.org/) as the Java Cryptography Extension (JCE) security provider. After the introduction of Android P, Android switched to using Google Conscrypt as the security provider. 
```

Comment:
What about changing file name to 2022-09-06-Umesh-GSoC-on-ConscryptMode.md or similar name?